scripts/configure.sh

```diff
@@ -344,7 +344,11 @@ if [ $skip_autogen == 0 ] ; then
     echo "running child instance of configure.sh in new autogen location..."
     "${autogen_path}/scripts/configure.sh" "${args_copy[@]}" -Z
 
-    echo "setting alias so ${build_cmd} runs autogen first on subsequent runs..."
+    echo "doing minimal build (just astrobee package) to force creation of devel/setup.sh..."
+    echo ${build_cmd} build astrobee
```

Comment:
This does not work on ROS2, here's the output:
```shell
➜  ros2_nasa_symlink_ws .astrobee/scripts/configure.sh -l -F -D

running autogen_ros_version_src.py to configure ROS version...
usage: autogen_ros_version_src.py [-h] [-v] [-r ROS_VERSION] [-d]
                                  [-c CHECKOUT_DIR] [-a AUTOGEN_DIR]
autogen_ros_version_src.py: error: autogen output dir /home/roque/Astrobee/src must not be a git checkout!
running child instance of configure.sh in new autogen location...
/home/roque/Astrobee/src/scripts/configure.sh: illegal option -- Z
configure.sh [-h] [-l <linux build>] [-a <arm build>] [-n <profile name>] [-p install_path] [-w workspace_path] [-B build_type] [-c <force clean cache>] [-C <don't clean cache>] [-d <with DDS>] [-D <without DDS>] [-r <with QP planner>] [-R <without QP planner>] [-f <with PicoFlexx driver>] [-F <without PicoFlexx driver>] [-k <use ccache if available>] [-K <do not use ccache>] [-v <with VIVE>] [-V <without VIVE>] [-t <with integration tests, requires gpu>] [-T <without integration test>] [-g <print debug information only>]
doing minimal build (just astrobee package) to force creation of devel/setup.sh...
colcon build astrobee
usage: colcon [-h] [--log-base LOG_BASE] [--log-level LOG_LEVEL]
              {build,extension-points,extensions,graph,info,list,metadata,test,test-result,version-check}
              ...
colcon: error: unrecognized arguments: astrobee
adding alias to devel/setup.sh so colcon runs autogen first on subsequent runs...
.astrobee/scripts/configure.sh: line 352: /home/roque/ros2_nasa_symlink_ws/devel/setup.sh: No such file or directory
(to suppress alias that invokes autogen, run \colcon instead of colcon.)
```

Reply:
Great point. My assumption going into this PR was that `configure.sh` was still a ROS1-only script that needed to be migrated to support ROS2 in some way, and I wasn't trying to complete that ROS2 migration myself. Sorry if that was an invalid assumption.

Under ROS1, that line of the script runs `catkin build astrobee`, which is a valid command to build only the Astrobee package. I just guessed that the same command might work with `colcon`, but apparently not. Probably there is a simple way to get the same effect with `colcon`, in which case that exact step would work under ROS2, but then the rest of the ROS1-only `configure.sh` script would presumably break :) .

But I hope it's ok to merge this PR and complete the ROS2 port of `configure.sh` later.

Reply:
I think this is fixed now?